source/docs/advanced/features/password-fields.md

```diff
@@ -0,0 +1,66 @@
+---
+id: password-fields
+title: Password fields
+---
+
+
+# Configure password validity
```

Comment:
issue: the main title is already used as a `h1`, so markdown links should start at the 2nd level

```suggestion
## Configure password validity
```
![screenshot-2021-12-06_10-34-09](https://user-images.githubusercontent.com/75968/144822423-24167efc-95e4-48ed-9763-98263e9e332a.jpg)